api/src/main/java/org/pmiops/workbench/user/UserAdminService.java

```diff
@@ -0,0 +1,61 @@
+package org.pmiops.workbench.user;
+
+import java.sql.Timestamp;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Optional;
+import java.util.Set;
+import org.pmiops.workbench.db.dao.UserEgressBypassWindowDao;
+import org.pmiops.workbench.db.model.DbUserEgressBypassWindow;
+import org.pmiops.workbench.model.EgressBypassWindow;
+import org.pmiops.workbench.utils.mappers.EgressBypassWindowMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserAdminService {
+  private final UserEgressBypassWindowDao userEgressBypassWindowDao;
+  private final EgressBypassWindowMapper egressBypassWindowMapper;
+  private final Clock clock;
+
+  @Autowired
+  public UserAdminService(
+      UserEgressBypassWindowDao userEgressBypassWindowDao,
+      EgressBypassWindowMapper egressBypassWindowMapper,
+      Clock clock) {
+    this.userEgressBypassWindowDao = userEgressBypassWindowDao;
+    this.egressBypassWindowMapper = egressBypassWindowMapper;
+    this.clock = clock;
+  }
+
+  public void createEgressBypassWindow(Long userId, Instant startTime, String description) {
+    Instant endTime = startTime.plus(2, ChronoUnit.DAYS);
```

Comment:
I assume 2 days is a requirement?

Reply:
Yup, it is from Moira's doc: https://docs.google.com/document/d/1hM1ODnAEzWQMrueXmDsYHePft5s9eLAsmq_Fh78D4Cg/edit?resourcekey=0-pZQ4jtyKEVnr7pezJ0oa7w
I will make this a constant.